content/syllabuses/web-dev-part-1.md

```diff
@@ -41,6 +41,7 @@ _db_id: 3
 - {{% contentlink path="topics/js-and-node-specific/node" %}}
 - {{% contentlink path="language-agnostic/validate-id-number" flavour="javascript" %}}
 - {{% contentlink path="sql/shop-project" flavour="" %}}
+- {{% contentlink path="projects/tdd/string-calculator-part-1" flavour="typescript" %}}
```

Comment:
I'm not entirely sure about this content link. Why is string calculator the only affected project?